Examples/Io/Performance/ActsExamples/Io/Performance/CKFPerformanceWriter.cpp

```diff
@@ -165,25 +165,45 @@ ActsExamples::ProcessCode ActsExamples::CKFPerformanceWriter::writeT(
       }
       // Fill fake rate plots
       m_fakeRatePlotTool.fill(m_fakeRatePlotCache, fittedParameters, isFake);
+
+      // Use neural network classification for duplication rate plots
+      // Currently, the network used for this example can only handle
+      // good/duplicate classification, so need to manually exclude fake tracks
+      if (m_cfg.useMLTrackClassifier && !isFake) {
+        // create vector of input features for neural network
+        std::vector<float> inputFeatures(3);
```

Comment:
Optimization: you only need to create this temporary structure once per `writeT` call instead of for each trajectory iteration.